addons/payment/tests/test_multicompany_flows.py

```diff
@@ -69,3 +69,13 @@ def test_pay_logged_in_another_company(self):
         self.assertEqual(processing_values['currency_id'], self.currency.id)
         self.assertEqual(processing_values['partner_id'], self.user_company_a.partner_id.id)
         self.assertEqual(processing_values['reference'], self.reference)
+
+    def test_delete_token(self):
```

Comment:
Tests are way easier to deal with (especially when they fail) when their name is explicit.
```suggestion
    def test_archive_token_logged_in_another_company(self):
```

Reply:
Quick docstring too, please.